src/Cronofy/OAuthBuilder.cs

```diff
@@ -0,0 +1,85 @@
+namespace Cronofy
+{
+    using System;
+    using Cronofy.Requests;
+
+    /// <summary>
+    /// Builder for OAuthDetails.
+    /// </summary>
+    public class OAuthBuilder : IBuilder<AddToCalendarRequest.OAuthDetails>
```

Comment:
Should be `sealed`. Perhaps called `OAuthDetailsBuilder` as it's building `OAuthDetails`.